src/app/Shared/Services/NotificationChannel.service.tsx

```diff
@@ -107,10 +107,14 @@ export class NotificationChannel {
 
     const notificationsUrl = fromFetch(`${this.apiSvc.authority}/api/v1/notifications_url`)
       .pipe(
-        concatMap(resp => from(resp.json())),
+        concatMap(resp => {
+          if (resp.ok) return from(resp.json());
+          throw new Error(resp.status + ' ' + resp.statusText);
```

Comment:
Does `resp` still have a useful response body at this point, even if `!resp.ok`? I think Cryostat is probably still sending some additional info about the failure here, like the stack trace or its backend "cause" message for the exception, so that would be useful to include in the error log and notification if we can get it.

Reply:
Here's what it looks like with the response body included:

Notification:
![image](https://user-images.githubusercontent.com/41028544/135542355-56368d90-8d61-4e82-8811-2f1dce2f136f.png)

Window console:
![image](https://user-images.githubusercontent.com/41028544/135542446-f70343a7-29e7-4f5b-a615-8e5ba3f4073e.png)